### ISSUE TITLE
[ENG][Quest][Howling Fjord] The Fallen Sisters

### ISSUE THREAD

**Issue by Technowizard15** [Atlantiss/BugTracker#2183]
A bunch of "Chill Nymph" (http://www.wowhead.com/npc=23678/chill-nymph) are friendly and "Lurielle's Pendant" can't be used on them.

**Comment by Technowizard15** [comment]
Last time I tried the quest all "Chill Nymph" were friendly. It seems that they aren't reset before the respawn.

**Comment by Samineh** [comment]
Yeah I have the same problem, all of the Chill Nymphs are friendly and can't use the quest Item on them.

**Comment by Hallieth** [comment]
For those trying to do the quest - the issue is still ongoing :)

**Comment by coba999** [comment]
bump

**Comment by aendry** [comment]
still broken. the same issue

**Comment by aendry** [comment]
ok, there is a workaround... just aproach to a Chill Nymph near a Scarlet Ivy also and let the Scarlet Ivy attack you, that will turn the Chill Nymph hostile and then you can use the quest iteem to get them free.
